hamster_gtk/overview/dialogs.py

```diff
@@ -60,8 +62,6 @@ def __init__(self, parent, app, *args, **kwargs):
         self.main_box = self.get_content_area()
         self._daterange = self._get_default_daterange()
 
-        self._charts = False
```

Comment:
I am not a huge fan of class attributes. Why does it not suffice to have an instance attribute?